.travis.yml

```diff
@@ -21,7 +21,7 @@ env:
   - SCRIPT="./scripts/test-docker"
   - SCRIPT="./manage.py compilemessages" INSTALLPIP=yes
   - SCRIPT="docker run -v $(pwd):/squad --workdir /squad fsfe/reuse reuse lint"
-  - SCRIPT="python3 -m pytest" INSTALLPIP=yes
+  - SCRIPT="python3 -m pytest" DATABASE=ENGINE=django.db.backends.postgresql_psycopg2:NAME=squad:USER=postgres:PASSWORD=squad:HOST=127.0.0.1:PORT=5432 INSTALLPIP=yes
```

Comment:
why are you only changing this test, but not the others? (sqlite is the default after all)

Reply:
This one uses pytest, and thought it would be easier to run pytest against postgres than find out how to exclude that test in pytest